wafer/talks/urls.py

```diff
@@ -10,7 +10,7 @@
 
 talks_router = router.register(r'talks', TalksViewSet)
 talks_router.register(
-    r'urls', TalkUrlsViewSet, basename='talks-urls',
+    r'urls', TalkUrlsViewSet, basename='talks-url',
```

Comment:
Should we make this `talk-urls` like the class name? It seems a bit odd to use url singular for a view that returns a list of URLs?

Reply:
So, I don't actually know what this is used for. Quite happy to drop that commit.